App/Containers/AddCaptionScreen.js

```diff
@@ -67,9 +65,12 @@ class AddCaptionScreen extends React.Component {
   }
 
   _renderAsset () {
+    const sourceUri = this.props.image.origURL && this.props.image.origURL !== ''
+      ? this.props.image.origURL
+      : this.props.image.uri
     return (
       <Image
-        source={{uri: this.props.image.origURL}}
+        source={{uri: sourceUri, isStatic: true}}
```

Comment:
android likes this isStatic parameter or just sits there and doesn't do anything and wont tell you why